base/linalg/lapack.jl

```diff
@@ -441,6 +486,91 @@ for (gebrd, gelqf, geqlf, geqrf, geqp3, geqrt, geqrt3, gerqf, getrf, elty, relty
     end
 end
 
+"""
+    gebrd!(A) -> (A, d, e, tauq, taup)
+
+Reduce `A` in-place to bidiagonal form `A = QBP'`. Returns `A`, containing the
+bidiagonal matrix `B`, `d`, containing the diagonal elements of `B`, `e`,
+containing the off-diagonal elements of `B`, `taup`, containing the
+elementary reflectors representing `P`, and `tauq` containing the
+elementary reflectors representing `Q`.
+"""
+gebrd!(A::StridedMatrix)
+
+"""
+    gelqf!(A, tau)
+
+Compute the `LQ` factorization of `A`, `A = LQ`. `tau` contains scalars
```

Comment:
lost in rebase https://github.com/JuliaLang/julia/commit/bfb08ad03e552207cc0ebb8d0d7b416548b63c70#commitcomment-13407758